portfolio/src/main/webapp/script.js

```diff
@@ -40,18 +44,52 @@ function showSlides() {
   for(slide of slides)
     slide.style.display = 'none';
 
-  slideIndex = slideIndex%slides.length; 
+  slideIndex = slideIndex % slides.length; 
   
   for(dot of dots)
     dot.className = dot.className.replace(" active","");
-
+  
```

Comment:
Nit: No trailing spaces on lines. At best, they create slightly larger diffs. At worst, they create really weird problems that are hard to track down.

Reply:
Fixed in PR 7